src/panels/config/integrations/integration-panels/zwave_js/zwave_js-provisioned.ts

```diff
@@ -0,0 +1,128 @@
+import { mdiDelete } from "@mdi/js";
+import { html, LitElement } from "lit";
+import { customElement, property, state } from "lit/decorators";
+import memoizeOne from "memoize-one";
+import { DataTableColumnContainer } from "../../../../../components/data-table/ha-data-table";
+import {
+  ZwaveJSProvisioningEntry,
+  fetchZwaveProvisioningEntries,
+  SecurityClass,
+  unprovisionZwaveSmartStartNode,
+} from "../../../../../data/zwave_js";
+import { showConfirmationDialog } from "../../../../../dialogs/generic/show-dialog-box";
+import "../../../../../layouts/hass-tabs-subpage-data-table";
+import { HomeAssistant, Route } from "../../../../../types";
+import { configTabs } from "./zwave_js-config-router";
+
+@customElement("zwave_js-provisioned")
+class ZWaveJSProvisioned extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @property({ type: Object }) public route!: Route;
+
+  @property({ type: Boolean }) public narrow!: boolean;
+
+  @property() public configEntryId!: string;
+
+  @state() private _provisioningEntries: ZwaveJSProvisioningEntry[] = [];
+
+  protected render() {
+    return html`
+      <hass-tabs-subpage-data-table
+        .hass=${this.hass}
+        .narrow=${this.narrow}
+        .route=${this.route}
+        .tabs=${configTabs}
+        .columns=${this._columns(this.narrow)}
+        .data=${this._provisioningEntries}
+      >
+      </hass-tabs-subpage-data-table>
+    `;
+  }
+
+  private _columns = memoizeOne(
```

Comment:
Can we show the date when a device was provisioned ?

Reply:
If the backend adds that info to the provisioning entry, yes

Reply:
Right so not this PR.

@raman325 / @MartinHjelmare is this something that we can add?

Reply:
The driver doesn't provide a date as far as I know. If we add it separately we'd need to keep track of this state somehow.

Reply:
that's correct, maybe something we can request as a feature to zwave-js

Reply:
If we are requesting features, maybe also allow the user to set a name for when it comes online?

Reply:
I think we can add anything we want to the provisioning entry?

https://github.com/zwave-js/node-zwave-js/blob/6ce51e47d7cd6d4419c0be4eb9fc676f38f0b088/packages/zwave-js/src/lib/controller/Inclusion.ts#L141